Add regex game test plans (#108)
Gamify-IT/docs: dev-manuals/test-plans/minigames/regex-game-frontend/plans/u-regex-game-2-highscore-update.md

# Highscore update test (`u.regex-game-2`)

Version: V1.0, 07.08.2022 \
Author: Leon Layer \
Tester: -

## Description

Tests if the high score increases correctly when a higher score is reached.

## Precondition

The game is in a state where the maximum possible higscore has not yet been reached.

## Postcondition

by playing, the current higscore will be exceeded and the higscore will be updated in the menu. 

## Typical procedure

> 1. the tester plays the game but does not yet reach the maximum higscore. 

Reviewer: ```suggestion
1. the tester plays the game but does not yet reach the maximum highscore. 
```

2. the tester sees his previous higscore in the menu. 
3. by trying again, he surpasses his previous best performance.

> 4. This will update the higscore in the meu.

Reviewer: ```suggestion
4. This will update the higscore in the menu.
```

> 1. the tester plays the game but does not yet reach the maximum higscore. 
> 2. the tester sees his previous higscore in the menu. 
> 3. by trying again, he surpasses his previous best performance.
> 4. This will update the higscore in the meu.

Reviewer: ```suggestion
1. the tester plays the game but does not yet reach the maximum highscore. 
2. the tester sees his previous highscore in the menu. 
3. by trying again, he surpasses his previous best performance.
4. This will update the highscore in the menu.
```

Reviewer 2: ```suggestion
1. Play the game without reaching the highscore 
2. Inspect your (previous) highscore in the menu 
3. Play again, now surpassing the highscore
4. The new highscore is displayed in the menu
```

## Alternative procedures

> 1.1. In the menu a wrong score is displayed as highscore \
> 2.1 The tester sees his previous higscore in the menu (0). 
> 4.1 The higscore
> 

Reviewer: ```suggestion
```

Reviewer 2: 4.1. isn't even English?
